app/assets/stylesheets/WIPbeertiles.css

```diff
@@ -0,0 +1,3 @@
+.beerTile {
```

Comment:
Is the "WIP" part of this file name short for "Work In Progress"? If so, please remove -- renaming files partway through your project can make it difficult to track changes, and it's better to store that info in a comment if necessary. However, since your whole app is a WIP right now, no need to note it at all :) 

If WIP doesn't stand for Work In Progress, please ignore!